code/rdb/rdbstandard.q

```diff
@@ -1,5 +1,5 @@
 // Get the relevant RDB attributes
-.proc.getattributes:{`date`tables!(.rdb.rdbpartition[],();tables[])}
+.proc.getattributes:{`partition`tables!(.rdb.getpartition[],();tables[])}
```

Comment:
Why does .rdb.rdbpartition need changed? Doesn't .rdb.getpartition just return .rdb.rdbparititon?

Reply:
not sure honestly, i just know that it throws a type error with .rdb.rdbpartition[] when int partitioned. I thought it was because .rdb.rdbpartition is a variable and its being invoked as if its a function? but then it works with date. getpartition[] works in both cases though so i went with it